force-app/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -0,0 +1,753 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Profile xmlns="http://soap.sforce.com/2006/04/metadata">
+    <applicationVisibilities>
+        <application>Events</application>
```

Comment:
@t-gul @bmtemakina there are permissions in the profile file which are not relevant to this ticket